adapters/quito.js

```diff
@@ -0,0 +1,139 @@
+'use strict';
+
+import { REQUEST_TIMEOUT } from '../lib/constants';
+import { default as baseRequest } from 'request';
+import { default as moment } from 'moment-timezone';
+
+exports.name = 'quito';
+
+const locations = [
+  'CML',
+  'COT',
+  'GUA',
+  'TUM',
+  'CH',
+  'P',
+  'ITC',
+  'CAR',
+  'JIP',
+  'BEL',
+  'CEN'
+];
+const measurementTypes = ['PM 2.5', 'PM 10', 'CO', 'NO2', 'SO2', 'O3'];
+
+const request = baseRequest.defaults({ timeout: REQUEST_TIMEOUT });
+
+const renameParameter = function (parameter) {
```

Comment:
You may be able to use the util function [unifyParameters](https://github.com/openaq/openaq-fetch/blob/develop/lib/utils.js#L59) here